### PR TITLE
Adds test for 4.3.6.

### PR DESCRIPTION
**Pull request by dbernstein**
Resolves: https://github.com/fcrepo/Fedora-API-Test-Suite/issues/141

### CODE REVIEW

File: src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpDelete.java [hunk 1]
```diff
@@ -53,6 +58,25 @@ public void ldpcvThatAdvertisesDeleteShouldRemoveContainerAndMementos(final Stri
...
+        if (hasHeaderValueInMultiValueHeader("Allow", "DELETE", optionsResponse)) {
+            //delete the ldpcv
+            final Response deleteResponse = doDelete(timeMap);
+            assertEquals("Timemap DELETE command failed.", deleteResponse.getStatusCode(), 204);
+            //verify that ldpcv is gone
+            final Response timeMapGet = doGetUnverified(timeMap);
+            assertEquals("Timemap was not deleted", timeMapGet.getStatusCode(), 404);
+            //verify that versioning model is gone
+            final Response getResponse = doGet(versionedResource);
+            confirmAbsenceOfLinkValue(ORIGINAL_RESOURCE_LINK_HEADER, getResponse);
```
Comment (awoods) [inline src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpDelete.java]: We should probably have an `else` block that adds a "Not Supported" result to the report.
For now, a `TODO: <new GitHub issue>` may be sufficient.

File: src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpDelete.java [hunk 2]
```diff
@@ -53,6 +58,25 @@ public void ldpcvThatAdvertisesDeleteShouldRemoveContainerAndMementos(final Stri
...
+        final String versionedResource = getLocation(response);
+        final String timeMap = getTimeMapUri(response).toString();
+
+        final Response optionsResponse = doOptions(timeMap);
+
+        //check that delete is supported
+        if (hasHeaderValueInMultiValueHeader("Allow", "DELETE", optionsResponse)) {
+            //delete the ldpcv
+            final Response deleteResponse = doDelete(timeMap);
+            assertEquals("Timemap DELETE command failed.", deleteResponse.getStatusCode(), 204);
```
Comment (awoods) [inline src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpDelete.java]: The `204` assertion already occurs in ``doDelete().

File: src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpDelete.java [hunk 3]
```diff
@@ -53,6 +58,25 @@ public void ldpcvThatAdvertisesDeleteShouldRemoveContainerAndMementos(final Stri
                                         "https://fcrepo.github.io/fcrepo-specification/#ldpcv-delete",
                                         ps);
 
+        //create a versioned resource
+        final Response response = createVersionedResource(uri, info);
+        final String versionedResource = getLocation(response);
+        final String timeMap = getTimeMapUri(response).toString();
```
[the remark this answers is not in the document]
Comment (dbernstein) [reply src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpDelete.java]: ++

File: src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpDelete.java (hunk 1 above)
Comment (dbernstein) [reply src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpDelete.java]: @awoods : Are you sure we want to do this?  It seems to me that perhaps there is a missing "MAY" test (as in "LDPCv may support DELETE" that will fail if delete is not supported.  I believe that is how were are currently doing it with other MAY tests.

File: src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpDelete.java (hunk 2 above)
Comment (dbernstein) [reply src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpDelete.java]: ++

**Pull request merged**